i18n/ads-language-pack-de/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 All notable changes to the "vscode-language-pack-de" language pack will be documented in this file.
 
 ## [Released]
+* March 31, 2021 - Release for VS Code 1.55
```

Comment:
I guess it doesn't matter too much but it'd be nice not to have a bunch of stuff referencing VS Code in our language packs...